R/input-slider.R

```diff
@@ -296,6 +298,37 @@ findStepSize <- function(min, max, step) {
   }
 }
 
+# Throw a warning if ever `value` is not in the [`min`, `max`] range
+validate_slider_value <- function(min, max, value, fun) {
+  if (is.null(min)   || is_na(min) ||
+      is.null(max)   || is_na(max) ||
+      is.null(value) || is_na(value))
+  {
+    stop(call. = FALSE,
+      sprintf("In %s(): `min`, `max`, and `value` cannot be NULL or NA.", fun)
+    )
+  }
+
+  if (!isTRUE(min(value) >= min)) {
```

Comment:
`value` can be a vector. If of size = 2, then it would always fail.

```r
❯❯ isTRUE(c(T, T))
[1] FALSE
```

```suggestion
  if (any(min(value) < min)) {
```

Reply:
`value` is passed to `min()`, so it returns just a single value.